src/bun.js/webcore/streams.zig

```diff
@@ -4244,27 +4242,27 @@ pub const File = struct {
 
         if (file.pathlike != .path and !(file.is_atty orelse false)) {
             if (comptime Environment.isWindows) {
-                bun.todo(@src(), {});
```

Comment:
why remove `bun.todo`? It logs where it happened

Reply:
it, and many others, were converted to `@panic("TODO on Windows")`, which gives a full trace and stops execution, which is more desired in this case because by ignoring the error you get into weird state like the zero length buffer. not as relevant now as the todo log is no longer disabled by default, but i like having these full-stop panic todos in these places.